src/actions/recomputeReduxState.js

```diff
@@ -327,7 +327,7 @@ const modifyControlsStateViaTree = (state, tree, treeToo, colorings) => {
       /* check mutations */
       if (node.branch_attrs && node.branch_attrs.mutations) {
         const keys = Object.keys(node.branch_attrs.mutations);
-        if (keys.length > 1 || (keys.length === 1 && keys[0]!=="nuc")) aaMuts = true;
+        if (keys.length > 1 || (keys.length === 1 && keys[0] !== "nuc")) aaMuts = true;
```

Comment:
There are quite a few stylistic changes such as these in this PR. I'm not necessarily against them, but would be easier to understand if they were different PRs as they are unrelated to narratives.

Reply:
Yeah thats my editor autoformatting everything xD
Will clean it up xD